plutus-playground-server/src/Playground/Interpreter.hs

```diff
@@ -81,11 +81,10 @@ mkCompileScript script =
   replaceModuleName script
     <> Text.unlines
       [ "",
-        "$ensureIotsDefinitions",
         "$ensureKnownCurrencies",
         "",
         "main :: IO ()",
-        "main = printSchemas (schemas, registeredKnownCurrencies, iotsDefinitions)"
+        "main = printSchemas (schemas, registeredKnownCurrencies)"
```

Comment:
Can you/have you checked that this still compiles through the frontend? It should do, and it should have been checked by the by the test suite, but I think it needs double-checking.

Reply:
I did this PR component-wise and always tried compiling and running tests. But seems like @michaelpj already pointed out that the tests are in fact _not_ passing so i need to look into that.

Reply:
Yes, thanks - was failing indeed. Fixed now. See below.